packages/kbn-grid-layout/grid/grid_height_smoother.tsx

```diff
@@ -42,23 +42,34 @@ export const GridHeightSmoother = ({
         dimensions.height ?? 0,
         smoothHeightRef.current.getBoundingClientRect().height
       )}px`;
+      smoothHeightRef.current.style.userSelect = 'none';
     });
 
+    const paddingSubscription = gridLayoutStateManager.runtimeSettings$
+      .pipe(
+        map(({ gutterSize }) => gutterSize),
+        distinctUntilChanged()
+      )
+      .subscribe((gutterSize) => {
+        if (!smoothHeightRef.current) return;
+        smoothHeightRef.current.style.padding = `${gutterSize}px`;
+      });
```

Comment:
Doing this through subscription is technically overkill, since guttersize cannot **currently** change - but it was done in future preparation for if we ever decide to make this a user-controlled value.